products/editions/enterprise-edition/b2b-suite/guides/core/rest-api.md

```diff
@@ -0,0 +1,101 @@
+# REST api
+
+You can download a plugin showcasing the topic [here](https://docs.enterprise.shopware.com/exampleplugins/B2bRestApi.zip).
+
+We use swagger.io for the documentation of our B2B-Suite endpoints. The created [swagger.json](https://gitlab.com/shopware/shopware/enterprise/b2b/-/blob/minor/swagger.json) file can be displayed with [swagger ui](http://swagger.io/swagger-ui/).
+
+## Table of contents
+* [Description](#description)
+* [A Simple Example](#a-simple-example)
+* [Adding the route](#adding-the-route)
+* [Complex routes](#complex-routes)
+
+## Description
+The B2B-Suite comes with its own extension to the REST-API. Contrary to Shopware own implementation that makes heavy use of the Doctrine ORM the B2B-Suite reuses the same services defined for the Storefront and therefore provides a controller format that is more reminiscent of Symfony 2.
```

Comment:
Please remove "and therefore provides a controller format that is more reminiscent of Symfony 2." This is not good :D